src/main/webapp/dev/js/common/scrollTo.es6

```diff
@@ -48,9 +48,9 @@ function scrollToElement(element, opts) {
         return;
     }
 
-    const navbar = document.getElementsByClassName('navbar')[0];
+    const navbar = $('#navbar')[0];
```

Comment:
Be careful, this one is `ByClassName`. The selector for class name is `.className` instead of `#className`.